Add media-query and desktop version HTML/CSS

Kanyorok opened the pull request:
- Add transition and animation to buttons and social icons on the site. 
-Add media queries to the site to make it responsive.

Kanyorok committed:
Add media-query and desktop version HTML/CSS

Kanyorok committed:
Fix portfolio responsiveness desktop version HTML/CSS

Kamba56 commented:
Hi @Kanyorok 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that **doesn't have a complete readme.** Your readme is missing the license section, kindly add that.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

Kanyorok committed:
Add README complete file

Add license changes to README file.

Kanyorok committed:
Add License file to branch

The license file is added to the current branch to meet the legal requirements.

Kanyorok committed:
Merge branch 'main' into desktop_transition_mediaquery_HTML/CSS

Kanyorok committed:
Update Details on README file

Add complete README file for the project

Yerimah commented on style.css:
```diff
@@ -1,89 +1,746 @@
+    flex-direction: column;
+  }
+
+  .contact_content p {
+    margin-top: 0;
+    font-family: "inter", sans-serif;
+    font-weight: 700;
+    font-style: normal;
+    font-size: 32px;
+    line-height: 138%;
+    text-align: center;
+    color: #172b4d;
+    margin-bottom: 119px;
+  }
```
- [ ] Hey @Kanyorok, Kudos on implementing the project requirements but I would like you to make some adjustments to this section so that your work looks exactly like your chosen Figma template.  make `margin-top: 13px; ` `font-size: 28px;`

> Figma design
<img width="671" alt="Screenshot 2023-05-06 at 13 16 03" src="https://user-images.githubusercontent.com/71140133/236623972-402d0ec8-64d2-4037-84d0-7166a5201d52.png">

> Your design
<img width="1099" alt="Screenshot 2023-05-06 at 13 15 29" src="https://user-images.githubusercontent.com/71140133/236624002-47101433-2f39-4a10-adee-94e257fa01a6.png">

Yerimah commented on index.html:
```diff
@@ -1,383 +1,416 @@
+          <span> My Recent Works </span>
+          <hr />
+        </header>
+        <article class="works_content">
+          <div class="image_placement">
+            <img src="icons/Works_cont.svg" alt="The Banner" />
+          </div>
+          <div class="text_placement">
```
- [ ] This element should be wrapped in a `section` tag and not an `article` tag. The section tag defines a section in a document. A requirement for this project is to use the section tag, not an article tag. kindly remove the article tags to improve your UI.
- [ ] Also try to make sure this section looks exactly like the original design by ensuring you adjust the font size, margins, and padding so that it aligns with the template provided.


> Figma design
<img width="644" alt="Screenshot 2023-05-06 at 13 11 32" src="https://user-images.githubusercontent.com/71140133/236624667-e515dc99-1bb0-4a1d-89ed-3bad166aad2b.png">

> Your design
<img width="1092" alt="Screenshot 2023-05-06 at 13 11 46" src="https://user-images.githubusercontent.com/71140133/236624682-3fcba74a-daf9-4926-90a5-1cd6f86b19c7.png">

Yerimah commented on index.html:
```diff
@@ -1,383 +1,416 @@
-        <button type="button">See project</button>
-      </article>
-    </section>
+      <section class="works">
+        <header class="works_title">
+          <span> My Recent Works </span>
+          <hr />
+        </header>
```
- [ ] I would suggest that you wrap this subheading in a `hx` tag and not a header tag because the `<header>` HTML element represents introductory content, typically a group of introductory or navigational aids. Fix this to improve the semantics and accessibility of your website.

Yerimah reviewed (changes requested):
# STATUS: REQUIRED CHANGES ♻️ 

Hi @Kanyorok, 👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f4b70556d45436b5038554458344c37536b552f67697068792e676966](https://user-images.githubusercontent.com/71140133/236625330-77b794b8-d1a2-45ae-801a-95b3b16f297c.gif)



## Highlights
- No linter error 👍 
- Descriptive PR title 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Making your PR summary more descriptive.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Kanyorok committed:
Fix styling for html desktop version HTML/CSS

Kanyorok committed:
Fix merge conflict due to earlier change.

gemmen29 reviewed (approved):
Hi @Kanyorok, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

![](https://media.giphy.com/media/mCIjCgs3nWQWfJZvPA/giphy-downsized-large.gif)

## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
